connectors/es/sink.py

```diff
@@ -824,8 +839,15 @@ def ingestion_stats(self):
             )
         return stats
 
-    def fetch_error(self):
-        return None if self._extractor is None else self._extractor.fetch_error
+    def get_error(self):
+        return (
+            None
+            if self._extractor is None
+            else self._extractor.extractor_error
+            or self._extractor_error
+            or self._sink.sink_error
```

Comment:
To keep things consistent that should probably be also `self._sink.error`, if we decide to go with `error` in the `Extractor`

Reply:
I think it's nice to have different errors reported differently (speaking of error categorisation for me):

Sink errors are a category of "We were unable to upload data to Elasticsearch", so IngestionError.
Extractor errors are a category of "We failed to get data from 3rd-party for whatever reason", so ConnectorError or ThirdPartyError.

Would it be reasonable to split it here and later I add different handling into SyncJobRunner?

Reply:
How do you mean splitting it? They're already split (`_extractor.error` vs `_sink.error`). Are you wanting to just wrap those errors to that the source is clear when the message is displayed in kibana?

Reply:
Just making two methods - `sink_error` and `extractor_error` in this class should be good enough, and do the "or" logic handling in the caller of orchestrator (SyncJobRunner)

Reply:
Yeah let's have it in one place for now, makes sense!